solvers/spot5/reference_lookup/src/lookup.py

```
@@ -0,0 +1,135 @@
+    parser.add_argument("--solution-dir", required=True)
+    args = parser.parse_args()
+
+    case_dir = Path(args.case_dir)
+    solution_dir = Path(args.solution_dir)
+    config_dir = Path(args.config_dir) if args.config_dir else None
+
+    try:
+        spot_file = _find_spot_file(case_dir)
+        manifest = _load_manifest()
+        digest = _sha256(spot_file)
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)